ui/app/pages/settings/info-tab/info-tab.component.js

```diff
@@ -19,71 +20,71 @@ export default class InfoTab extends PureComponent {
           { t('links') }
         </div>
         <div className="info-tab__link-item">
-          <a
+          <Button
+            type="link"
             href="https://metamask.io/privacy.html"
             target="_blank"
             rel="noopener noreferrer"
+            className="info-tab__link-text"
           >
-            <span className="info-tab__link-text">
-              { t('privacyMsg') }
-            </span>
-          </a>
+            { t('privacyMsg') }
+          </Button>
         </div>
         <div className="info-tab__link-item">
-          <a
+          <Button
+            type="link"
             href="https://metamask.io/terms.html"
             target="_blank"
             rel="noopener noreferrer"
+            className="info-tab__link-text"
           >
-            <span className="info-tab__link-text">
-              { t('terms') }
-            </span>
-          </a>
+            { t('terms') }
+          </Button>
         </div>
         <div className="info-tab__link-item">
-          <a
+          <Button
+            type="link"
             href="https://metamask.io/attributions.html"
             target="_blank"
             rel="noopener noreferrer"
+            className="info-tab__link-text"
           >
-            <span className="info-tab__link-text">
-              { t('attributions') }
-            </span>
-          </a>
+            { t('attributions') }
+          </Button>
         </div>
         <hr className="info-tab__separator" />
         <div className="info-tab__link-item">
-          <a
+          <Button
+            type="link"
             href="https://support.metamask.io"
             target="_blank"
             rel="noopener noreferrer"
+            className="info-tab__link-text"
           >
-            <span className="info-tab__link-text">
-              { t('supportCenter') }
-            </span>
-          </a>
+            { t('supportCenter') }
+          </Button>
         </div>
         <div className="info-tab__link-item">
-          <a
+          <Button
+            type="link"
             href="https://metamask.io/"
             target="_blank"
             rel="noopener noreferrer"
+            className="info-tab__link-text"
           >
-            <span className="info-tab__link-text">
-              { t('visitWebSite') }
-            </span>
-          </a>
+            { t('visitWebSite') }
+          </Button>
         </div>
         <div className="info-tab__link-item">
-          <a
+          <Button
+            type="link"
             href="https://metamask.zendesk.com/hc/en-us/requests/new"
             target="_blank"
             rel="noopener noreferrer"
+            className="info-tab__link-text"
           >
-            <span className="info-tab__link-text">
-              { t('contactUs') }
-            </span>
-          </a>
+            { t('contactUs') }
+          </Button>
         </div>
       </div>
     )
```

Comment:
Above changes have centered and padded the About page's Links section

<details>
<summary>About page's link section</summary>
<img src='https://user-images.githubusercontent.com/13376180/90593520-fe8ca780-e19c-11ea-9176-7753e3640e03.png'>
</details>

Reply:
On it.

Reply:
<details>
<summary>Fixed</summary>

<img src="https://user-images.githubusercontent.com/4448075/90660339-68439a80-e20b-11ea-9ee7-44ddcaeceda0.png" />

</details>